index.js

```diff
@@ -199,13 +199,13 @@ var Base = Class.extend({
     }
 
     var pkSql = '';
-    if (primaryKeyColumns.length > 1) {
-      pkSql = this._handleMultiPrimaryKeys(primaryKeyColumns);
-
-    } else {
-
-      primaryKeyColumns[0] = primaryKeyColumns[0].name;
-      columnDefOptions.emitPrimaryKey = true;
+    if (primaryKeyColumns.length) {
```

Comment:
Ah ok now I understand, wasn't clear from the beginning.

But instead of doing this I suggest checking that length is greater than 0, as this makes more sense then checking if the variable is somehow initialized and the reader understands the intention.